ADOPTERS.MD

```diff
@@ -0,0 +1,11 @@
+# Knative Adopters
+
+This page contains a list of organizations who are using Knative in production or in stages of development.
+
+If you would like to be included in this table, please submit a comment to [this issue](https://github.com/knative/community/issues/688) and your information will be added.
+
+## Adopters
+
+| Organization | Status | More Information
```

Comment:
Just a nit (seems to display correctly in the rich view), but shouldn't the '|' before More Information be indented one more over and shouldn't there be a trailing '|'?

Reply:
I don't think it makes a difference, but let me update so it's not confusing